back-end-side/Controllers/UploadController.cs

```diff
@@ -14,20 +14,35 @@ namespace back_end_side.Controllers
     
     public class UploadController : ControllerBase
     {
+        public static readonly List<Record> RecordsFromFile = new List<Record>();
+
         [HttpPost]
         public ActionResult Post([FromForm] FileModel file)
         {
+            
+            Console.WriteLine("BANK: " + file.Bank);
+            ReportReader.bank = file.Bank;
             try
             {
                 if (file.FormFile != null && file.FileName != null)
                 {
-                    string path = Path.Combine(Directory.GetCurrentDirectory(), "uploadedFiles", file.FileName);
+                    string path = Path.Combine(Directory.GetCurrentDirectory(), "uploadedFiles", file.FileName); 
+                    string newPath = Path.Combine(Directory.GetCurrentDirectory(), "uploadedFiles", "report.csv");
+                   
                     using (Stream stream = new FileStream(path, FileMode.Create))
                     {
```

Comment:
You can delete curly brackets, they're irrelevant for _using_ statement

Reply:
Ok, will do